Source/WebCore/rendering/MotionPath.h

```diff
@@ -44,6 +45,7 @@ class MotionPath {
     static std::optional<MotionPathData> motionPathDataForRenderer(const RenderElement&);
     static bool needsUpdateAfterContainingBlockLayout(const PathOperation&);
     static void applyMotionPathTransform(const RenderStyle&, const TransformOperationData&, TransformationMatrix&);
+    WEBCORE_EXPORT static std::optional<Path> computePathForBox(const BoxPathOperation&, const TransformOperationData&);
```

Comment:
I don't think this needs WEBCORE_EXPORT, this isn't used outside of WebCore.

Reply:
This is needed due to the weirdness of WebCoreArgumentCoders.serialization.in since this function is used in the inline getPath.